nominal/cli/util/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .global_decorators import client_options, global_options
+
+__all__ = [
+    "client_options",
+    "global_options",
+]
```

Comment:
I would either define these functions in this init, or just leave it empty - it's not to be imported by any users so the additional flexibility of being able to do `from . import client_options` as opposed to e.g. `from ._options import client_options` is kinda w/e. we don't need to "export" them since it's all the same package iykwim

Reply:
ah ok. hm..will think about this a bit